internal/pkg/archer/project.go

```diff
@@ -31,3 +39,11 @@ type ProjectCreator interface {
 type ProjectGetter interface {
 	GetProject(projectName string) (*Project, error)
 }
+
+// ProjectResourceStore fetches resources related to the project.
+type ProjectResourceStore interface {
+	// Fetch all regional resources in a project.
+	GetRegionalProjectResources(project *Project) ([]*ProjectRegionalResources, error)
+	// Fetch the project regional resources for a particular region
```

Comment:
nit: can we have all comments end with a dot

Reply:
is that a style thing?

Reply:
Yeah: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

> Comments should begin with the name of the thing being described and end in a period

Reply:
WHAT IF THE COMMENT IS A QUANDARY.